### ISSUE TITLE
helmfiles include and charts in same repo

### ISSUE THREAD

**Issue by sstarcher** [roboll/helmfile#445]
Is it intended to ignore `charts` if you have a helmfiles include

```
helmfiles:
- helmfiles_blah.yaml

charts:
- blah
```

With the above helmfiles_blah.yaml will run and anything under charts will not be processed.

**Comment by mumoshu** [comment]
@sstarcher Good point! I think we had not documented that behavior at all.

This is expected at least to me, as you have no way to specify ordering between `helmfiles` and `charts`(or `releases` today).

Any reason you want to write your helmfile.yaml that way?

For example I have once considered to write up a proposal to allow inline helmfiles under the `helmfiles:` section. It would allow you to order sub-helmfiles, without separating your tiny helmfile.yaml into even more tiny sub-helmfiles just for ordering.

```yaml
helmfiles:
  - helmfiles_blah.yaml
  - charts:
    - blah
```

I'm just unsure if this actually helps anyone. I wanted to gather specific use-cases before going on :)

**Comment by sstarcher** [comment]
Nope the main reason was I was migrating to the new template format and started to split them out at finer details, but some did not fit so I needed a catch all.  

Essentially I previously had

```
* helmfile.yaml
  * account
    * dev
      * helmfile.yaml
    * stg
      * helmfile.yaml
    * prod/
      * helmfile.yaml
```

And moved to

```
* helmfile.yaml
  * account
    * dev
      * helmfile.yaml -> includes the teams helmfiles
      * teams
        * team1/helmfile.yaml
        * team2/helmfile.yaml
```

Where the helmfile.yaml under account/dev had a few stragglers that did not tit into the last one.  I have a few options.

* I can move those out to a different file.
* I can move the includes from the account/dev to the root helmfile
* helmfile.yaml could support loading the includes and then loading the charts after the includes or vice versa?

**Comment by mumoshu** [comment]
> helmfile.yaml could support loading the includes and then loading the charts after the includes

I prefer this, the option 3, which has been implemented in #522 😃
